.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       matrix:
         # supported python versions can be found here
         # https://github.com/actions/python-versions/releases
-        python-version: [3.6, 3.7, 3.8, 3.9, 3.10.0-beta.3]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Comment:
IIRC for "3.10" to work in YAML it has to be quoted (and is otherwise interpreted as a float - i.e. 3.1)

Reply:
Oh, a good point. Github is not running this anyway, I guess I need to flick some switch to make that happen too. Also looks like typing requires at least 3.8.

Reply:
Making any further changes in our main branch, sending one here was already quite accidental.